src/Twig/AppExtension.php

```diff
@@ -38,6 +41,7 @@
 use Twig\TwigFilter;
 use Twig\Environment as TwigEnvironment;
 use Twig\TwigFunction;
+use \EMS\CommonBundle\Search\Search as CommonSearch;
```

Comment:
```suggestion
use \EMS\CommonBundle\Search\Search;
```

Reply:
We also use EMS\CoreBundle\Entity\Form\Search;